scripts/storage-emulator-integration/tests.ts

```diff
@@ -1989,11 +1989,135 @@ describe("Storage emulator", () => {
       if (!TEST_CONFIG.keepBrowserOpen) {
         await browser.close();
       }
-      if (!TEST_CONFIG.useProductionServers) {
-        await test.stopEmulators();
+      if (TEST_CONFIG.useProductionServers) {
+        delete process.env.GOOGLE_APPLICATION_CREDENTIALS;
       } else {
+        await test.stopEmulators();
+      }
+    });
+  });
+
+  emulatorSpecificDescribe("Emulator Integrity", function (this) {
+    // eslint-disable-next-line @typescript-eslint/no-invalid-this
+    this.timeout(TEST_SETUP_TIMEOUT);
+
+    let testGcsBucket: Bucket;
+    let storage: Storage;
+
+    const initFirebaseSdkPage = async () => {
+      page = await browser.newPage();
+      await page.goto("https://example.com", { waitUntil: "networkidle2" });
+
+      await page.addScriptTag({
+        url: "https://www.gstatic.com/firebasejs/7.24.0/firebase-app.js",
+      });
+      await page.addScriptTag({
+        url: "https://www.gstatic.com/firebasejs/7.24.0/firebase-auth.js",
+      });
+      // url: "https://storage.googleapis.com/fir-tools-builds/firebase-storage-new.js",
```

Comment:
?

Reply:
good question, removed here and where I copied this from